Test CODEOWNERS

dan-knight opened the pull request:
Added a test `CODEOWNERS` file with myself and the software working group. This is not a permanent setup. For example, Paul should probably be included on the open source packages for which he's listed as the maintainer on CRAN.

tyamaguchi-ucla commented:
@dan-knight add @uclahs-cds/software-wg as admin. You should have permission to create branch protection rules.

dan-knight commented:
> @dan-knight I just added @uclahs-cds/software-wg as admin. You should have permission to create branch protection rules.

I added branch protection rules similar to our other repos. I'm surprised that there wasn't branch protection, actually!
